make/autoconf/toolchain.m4

```diff
@@ -696,8 +696,13 @@ AC_DEFUN_ONCE([TOOLCHAIN_DETECT_TOOLCHAIN_CORE],
   if test "x$TOOLCHAIN_TYPE" != xmicrosoft; then
     AS="$CC -c"
   else
-    # On windows, the assember is "ml.exe"
-    UTIL_LOOKUP_TOOLCHAIN_PROGS(AS, ml)
+    if test "x$OPENJDK_TARGET_CPU" = "xx86_64"; then
```

Comment:
Is the proper test for x86_64 or for 64-bit CPUs? Or phrased differently, what is the name of the executable on aarch64? The comment below states that the test is for word size, but the test itself is for CPU architecture.

Reply:
Changed to:
   if test "x$OPENJDK_TARGET_CPU_BITS" = "x64"; then